joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50

#Pozycja barku, przód tył - ustawiono
joint1_position_controller:
  type: effort_controllers/JointPositionController
  joint: Shoulder_01
  pid: {p: 62000.0, i: 20.0, d: 600.0}
  

#Pozycja barku, lewo prawo - ustawiono
joint2_position_controller:
  type: effort_controllers/JointPositionController
  joint: Shoulder_02
  pid: {p: 80000.0, i: 220.0, d: 1000.0}

#Martwy joint - bez ustawienia
joint3_position_controller:
  type: effort_controllers/JointPositionController
  joint: Arm_01
  pid: {p: 0.0, i: 0.0, d: 600.0}

#Łokieć, rotacja - ustawione
joint4_position_controller:
  type: effort_controllers/JointPositionController
  joint: Arm_02
  pid: {p: 50000.0, i: 20.0, d: 800.0}

#Łokieć, góra dół - ustawione
joint5_position_controller:
  type: effort_controllers/JointPositionController
  joint: Arm_03
  pid: {p: 42000.0, i: 100.0, d: 340.0}

#Nadgarstek, zgięcie - ustawione
joint6_position_controller:
  type: effort_controllers/JointPositionController
  joint: Wrist
  pid: {p: 50000.0, i: 120.0, d: 180.0}

#Nadgarstek, rotacja - ustawione
joint7_position_controller:
  type: effort_controllers/JointPositionController
  joint: Hand
  pid: {p: 16000.0, i: 380.0, d: 540.0}

#Palec wskazujący, część A - ustawione
joint8_position_controller:
  type: effort_controllers/JointPositionController
  joint: Finger_01_A
  pid: {p: 2000.0, i: 60.0, d: 200.0, antiwindup: true}

joint9_position_controller:
  type: effort_controllers/JointPositionController
  joint: Finger_01_B
  pid: {p: 2000.0, i: 60.0, d: 200.0}

joint10_position_controller:
  type: effort_controllers/JointPositionController
  joint: Finger_02_A
  pid: {p: 2000.0, i: 60.0, d: 200.0}

joint11_position_controller:
  type: effort_controllers/JointPositionController
  joint: Finger_02_B
  pid: {p: 2000.0, i: 60.0, d: 200.0}

joint12_position_controller:
  type: effort_controllers/JointPositionController
  joint: Finger_03_A
  pid: {p: 2000.0, i: 60.0, d: 200.0}

joint13_position_controller:
  type: effort_controllers/JointPositionController
  joint: Finger_03_B
  pid: {p: 2000.0, i: 60.0, d: 200.0}